src/worldcereal/openeo/inference.py

```diff
@@ -28,7 +28,7 @@ def dependencies(self) -> list:
         return []  # Disable the dependencies from PIP install
 
     def output_labels(self) -> list:
-        return ["classification", "probability"]
+        return ["classification", "max_probability"]
```

Comment:
when individual probabilities are not included, should we call this just `probability` for clarity? Or does this give issues downstream?
edit: see next comment, I'm confused.